protoc-gen-bq-json-schema/genjson/message.go

```diff
@@ -8,7 +8,13 @@ import (
 )
 
 func GenerateSchemaFile(gen *protogen.Plugin, msg *protogen.Message, config Config) error {
-	g := gen.NewGeneratedFile(fmt.Sprintf("%s.json", msg.Desc.Name()), "")
+	var filename string
+	if config.SchemaOptions.PrefixWithProtoPackage {
+		filename = string(msg.Desc.FullName())
```

Comment:
@niklaskb Should we just make this the default instead of making it configurable? It shouldn't be a breaking change to update file names anyway, right?

Reply:
Sure. Might be a breaking change depending on how you use the generated files, but should be easy to fix though.

Reply:
Top! IMO let's remove the option and make this the default - but to make sure we produce idiomatic file names, we should do the Go equivalent of `ToLower` and `ReplaceAll(".", "_")`